components/PersonView/PersonDetails.jsx

```diff
@@ -63,9 +63,9 @@ const PersonDetails = ({
                 <dt className="govuk-summary-list__key">Address</dt>
                 <dd className="govuk-summary-list__value">
                   {addressList &&
-                    Object.values(addressList[0]).map((addressLine) => (
-                      <p>{addressLine}</p>
-                    ))}
+                    Object.values(addressList[0])
```

Comment:
we should filter the `addressList` and get the one with value `Y` and then just show `addressLine1`.